src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -554,6 +554,23 @@ void submitStreamOperation(final PathOperation streamOp, final OpType opType) {
 		}
 	}
 
+	boolean completeStreamCreateOperation(final PathOperation streamOp, final boolean result, final Throwable thrown,
```

Comment:
Couldn't we get the scope/stream names from the `streamOp`? Just to not to pass extra arguments